gcloud-java-dns/src/main/java/com/google/gcloud/dns/Zone.java

```diff
@@ -61,8 +134,7 @@ public Zone(Dns dns, ZoneInfo zoneInfo) {
   public static Zone get(Dns dnsService, String zoneName, Dns.ZoneOption... options) {
     checkNotNull(zoneName);
     checkNotNull(dnsService);
-    ZoneInfo zoneInfo = dnsService.getZone(zoneName, options);
-    return zoneInfo == null ? null : new Zone(dnsService, zoneInfo);
+    return dnsService.getZone(zoneName, options);
```

Comment:
I think we decided not to have this method as it does not add any value to `dns.getZone`.